### DISCUSSION TITLE
Tree Supports in SuperSlicer

### DISCUSSION THREAD

**Discussion by nivekmai** [supermerill/SuperSlicer#1944] [Community contribution]
Adding here since I don't see any discussions for it, and there's one extra trick needed specifically for SuperSlicer.

You can add tree supports in SuperSlicer by using meshmixer, there's [a pretty good tutorial on youtube](https://www.youtube.com/watch?v=n9SOT7z9M6M).

The trick with SuperSlicer is that you need to first turn off "Auto-center parts" in Configuration -> Preferences, and then make sure to export the meshes as 3mf from meshmixer. If you do both of those, it should import them in the right alignment off to the side of the build plate just like what the video shows happening in Simplify3d. Then just make sure you're moving both parts together whenever setting things up (and don't click auto-arrange, or you'll lose your alignment).

**Comment by supermerill** [comment]
related issue: #27

**Comment by jlipavsky** [comment]
now that this is a actual thing in Prusa slicer should we be expecting it to make its way over?

**Comment by RandomGgames** [comment]
^^^

**Comment by VidiotGeek** [comment]
^ bump. Anyone know the status of this?

**Reply by nivekmai** [reply]
supermerill is working on updating to prusa 2.6 which should bring tree supports to to SS

**Reply by jlipavsky** [reply]
Do we have a time approximation?  soon I'm hoping......

**Reply by VidiotGeek** [reply]
I'm just happy to hear there's progress in that direction. Sooner would be great, but it's free software.

**Reply by jeremytodd1** [reply]
I wouldn't hold your breath. The last full update for SuperSlicer was just shy of a year (Sept 23,2022). 

SuperSlicer is still missing features from Prusa version 2.5, which is quite old at this point. Honestly, I don't see all the 2.6 features ever coming to SuperSlicer at this point. 

Development for SuperSlicer is essentially dead and I believe that if people want to continue using SuperSlicer, someone needs to create a fork so progress can be made again. I'm grateful for what Supermerril has made so far with Superslicer, but the time has come where change needs to happen. SuperSlicer is missing too many features at this point.

In the meantime, I recommend OrcaSlicer. Otherwise, PrusaSlicer is also good. Once you make the switch you wont look back.

**Reply by VidiotGeek** [reply]
OrcaSlicer looks very interesting... I'll give PrusaSlicer and OrcaSlicer a try. Since they all share more or less the same base, I'm hoping that my settings, filament profiles and machine profiles will all transfer over with few or no problems?

Much easier said than done--maybe some of the fork developers combine efforts and merge back into (fewer) forks? All the same, thanks to @supermerill et al for the great work they've done.

**Reply by oldmanbluntz** [reply]
And in 2025 we are still waiting for superslicer 2.7 with tree suppert

**Reply by EchoAlpha519** [reply]
> And in 2025 we are still waiting for superslicer 2.7 with tree suppert

You may be interested in [PrusaSlicer BOSS (Best of SuperSlicer)](https://github.com/mjonuschat/PrusaSlicer). It takes the latest release of PrusaSlicer and adds the "best"* features from SuperSlicer.

*"Best" according to the maintainer of the repository. I haven't used it enough to see what features are there/missing.

**Reply by oldmanbluntz** [reply]
I just switched over to PrusaSlicer entirely. It actually does a few other things I had hoped superslicer would do, like allow me to add some of the other flex filaments to my ender 3 S1 profile, instead of just creality filaments, and the organic tree supports.
